src/app/app.component.ts

```diff
@@ -63,7 +61,8 @@ export class AppComponent implements OnInit, OnDestroy {
     filter(item =>
          item.status === ItemStatus.DOWNLOADING
       || item.status === ItemStatus.UPLOADING
-      || (item.status === ItemStatus.PENDING && Type.EXPORT_GCP));
+      || (item.status === ItemStatus.PENDING && Type.EXPORT_GCP
+      || item.status === ItemStatus.EXPORTING_S3));
```

Comment:
I think the `Type.EXPORT_GCP` is incorrect (from before), shouldn't it be `(item.status === PENDING && item.type == Type.EXPORT_GCP)`. Also it looks like the parens are around the wrong subexpression.